BUILD.md

```diff
@@ -1,5 +1,25 @@
 # Build Instructions
 
+The following dependencies are required for this project:
+
+- **Git**
+- **Zip** (for [vcpkg](https://vcpkg.io))
+- **Curl** (for [vcpkg](https://vcpkg.io))
+- **Tar** (for [vcpkg](https://vcpkg.io))
+- **Ninja-build** (for [vcpkg](https://vcpkg.io))
+- **Pkg-config**
+
+## Compilation steps
+
+1. **Installing Dependencies on Debian**
+
+    To install the necessary dependencies on a Debian-based system, run the following commands:
+
+    ```bash
+    sudo apt-get update
+    sudo apt-get install git zip curl tar ninja-build pkg-config
+    ```
+
 1. **Clone the Repository**
```

Comment:
Number are not OK, starting from this one.

Reply:
😅 it was on purpose. Markdown renders the number incrementally despite this. But I can change it of course.

Reply:
Done